lasagne/layers/conv.py

```diff
@@ -841,6 +846,11 @@ class TransposedConv2DLayer(BaseConvLayer):
         which of the possible output shapes to return when stride > 1.
         If not specified, the smallest shape will be returned.
 
+    n : int or None
+        The dimensionality of the transposed convolution (i.e., the number of
+        spatial dimensions of each feature map and each convolutional filter).
+        If ``None``, will be inferred from the input shape.
+
```

Comment:
The `untie_biases` docstring still needs to be adapted (https://github.com/Lasagne/Lasagne/pull/850/files#r123304501). Unfortunately I cannot comment on the correct line for that because it's not part of the diff.